roles-in-tech/security.md

```diff
@@ -17,16 +19,22 @@
 
 Apprentices will learn secure development basics, common pitfalls, and how to avoid them.
 
+### What employers are looking for
+
+- You'd have to understand things like XSS, CSRF, SQL injections, input validation, etc.
```

Comment:
Please change this section's tone to be less conversational/ more like the rest of your outline. Also, avoid using all caps and consider *italics* or **bold** instead.

Reply:
Also, remember to resolve your merge conflicts once this is ready.